### DISCUSSION TITLE
Request HuggingFace

### DISCUSSION THREAD

**Discussion by silgon** [kardolus/chatgpt-cli#100] [Ideas]
I just found out about the huggingface API (https://huggingface.co/playground?modelId=meta-llama/Llama-3.3-70B-Instruct). And well I just thought about this amazing package, I don't know if it is supported, I didn't see it in the read me. So, this is a feature request for the amazing @kardolus . Hope you are doing fine, I still use your package in a daily basis in my command line. Actually, when I don't get the result or I need something more precise, I go to huggingchat. So, I was thinking that It would be great to have the implementation altogether.
Best of luck to you and thanks again for the amazing package! =D

**Comment by kardolus** [comment]
Hi @silgon, thanks for the kind works. Llama should work; I tested the CLI with a Llama instance at work. I will take a look at huggingface, looks like a cool project!
